modules/cluster-logging-collector-external.adoc

```diff
@@ -83,7 +157,7 @@ To configure the *out_forward* plug-in:
   transport tls <3>
 
   tls_verify_hostname true <4>
-  tls_cert_path /etc/ocp-forward/ca_cert.pem <5>
+  tls_cert_path <path to file> <5>
```

Comment:
tls_cert_path <path to file> <5>. Have you explain where the path from? It should be /etc/ocp-forward/ + 'arbitrary-name-of-key1' from secury-forward secret.  

How about adjust the sequnce?  create secure-forward secret at first. and then create secure-forward confimap.